partner_contact_in_several_companies/models/ir_actions.py

```diff
@@ -0,0 +1,24 @@
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from odoo import models
+
+
+class IRActionsWindow(models.Model):
+    _inherit = "ir.actions.act_window"
+
+    def read(self, fields=None, context=None, load="_classic_read"):
+        actions = super(IRActionsWindow, self).read(fields=fields, load=load)
+        for action in actions:
+            if action.get("res_model", "") == "res.partner":
+                # By default, only show standalone contact
+                action_context = action.get("context", "{}")
+                if "search_show_all_positions" not in action_context:
+                    action["context"] = action_context.replace(
+                        "{",
+                        (
+                            "{'search_show_all_positions': "
+                            "{'is_set': True, 'set_value': False},"
+                        ),
+                        1,
+                    )
```

Comment:
**nitpick:** I think that this needs to be a little more robust

Reply:
I agree but I think this is out of the scope of this migration.

I think we could proceed with the migration and then fix the problem on `16.0` and then port it on `14.0`